### PR TITLE
checking correct versions of gloo and solo-kit

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: ci/check-code-and-docs-gen.sh [hunk 1]
```diff
@@ -35,10 +35,11 @@ if [[ $(git status --porcelain | wc -l) -ne 0 ]]; then
   exit 1;
 fi
 
-if [[ $(git --git-dir=/workspace/gopath/src/github.com/solo-io/supergloo/.git --work-tree=/workspace/gopath/src/github.com/solo-io/supergloo status --porcelain | wc -l) -ne 0 ]]; then
-  echo "Generating code produced a non-empty diff in the gloo repo"
-  echo "Make sure the go_import directory in protos is set to directory in supergloo, not in gloo."
+if [[ $(git --git-dir=/workspace/gopath/src/github.com/solo-io/gloo/.git --work-tree=/workspace/gopath/src/github.com/solo-io/gloo status --porcelain | wc -l) -ne 0 ]]; then
+echo "Generating code produced a non-empty diff in the gloo repo"
```
Comment (rickducott) [inline ci/check-code-and-docs-gen.sh]: supergloo

File: changelog/v0.3.3/check-versions.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+changelog:
+  - type: NON_USER_FACING
```
Comment (rickducott) [inline changelog/v0.3.3/check-versions.yaml]: dependency bump

**Comment by EItanya** [comment]
==MERGE_WHEN_READY==

**Pull request merged**